content_scripts/vimium_frontend.coffee

```diff
@@ -229,10 +235,13 @@ getActiveState = ->
 #
 # The backend needs to know which frame has focus.
 #
-registerFocus = ->
-  # settings may have changed since the frame last had focus
-  settings.load()
-  chrome.runtime.sendMessage({ handler: "frameFocused", frameId: frameId })
+registerFocus = (event) ->
+  if event.target == window
+    # Settings may have changed since the frame last had focus.
+    settings.load()
+    # Don't register frameset containers; focusing them is no use.
+    unless document.body?.tagName.toLowerCase() == "frameset"
```

Comment:
This mean that focusing a `<frameset>` will leave the Vomnibar visible, no? Is this the right behaviour?